lib/web_console/railtie.rb

```diff
@@ -10,7 +10,11 @@ class Railtie < ::Rails::Railtie
       require "bindex"
       require "web_console/extensions"
 
-      ActionDispatch::DebugExceptions.register_interceptor(Interceptor)
+      # Interceptor references ActionView::Template::Error.
```

Comment:
`ActionView::Template::Error` is defined inside the method definition so it should not be a problem. Why do you think this need to changed given the method body is not executed at load?

Reply:
Ah, sorry. I was basing it on comments by @Edouard-chin  in https://github.com/rails/rails/pull/38024. I'll see if I can reproduce his errors with his branch.

Reply:
The problem was that `DebugView` where loaded inside `ActionDispatch::DebugExceptions`. It is not anymore since https://github.com/rails/rails/commit/07ec8062e605ba4e9bd153e1d264b02ac4ab8a0f, so I think the problem is fixed.

Reply:
Ah nice! I'll close this one.
Is https://github.com/rails/rails/pull/38024 still something worth pursuing?

Reply:
In the end, is this PR needed or should we close it?

Reply:
The problem was indeed coming from web-console, but the the real underlying issue was in Rails. That has been fixed in https://github.com/rails/rails/pull/38026 so we can close this PR. Thanks for looking at this @p8 though ❤️